lib/flame_splash_screen.dart

```diff
@@ -1,5 +1,6 @@
 library flame_splash_screen;
 
+import 'package:flutter/material.dart';
```

Comment:
Just wondering if that import is been used, there isn't anymore change on this file despite of it.